src/alert_distributor.coffee

```diff
@@ -48,6 +48,7 @@ module.exports = class AlertDistributor
   # Parse out event data from StatsD packet
   parsePacket: (packet) ->
     for event in packet.toString().split('\n') or []
+      continue if !event
```

Comment:
thanks! well i hacked the js output to make it work and then tried my best to write coffee :8ball: